preprocess-and-train.py

```diff
@@ -10,6 +10,7 @@
 # pre-processing on the fly is possible because this is small project and
 # not a lot of images will be there as well.
 def preprocess(img):
+    img = cv2.resize(img,(225,225))
```

Comment:
what other preprocessing techniques did you try?
and what were the results of those techniques?

Reply:
We haven't tried any other preprocessing techniques and the accuracy we got was more than 99%. But we can try other techniques and compare the accuracy.

Reply:
Yes, please do that so that you can learn about them. I am closing this review in the hopes that you will take a look at them and learn.

Reply:
Absolutely!